sql/core/src/main/scala/org/apache/spark/sql/execution/HiveResult.scala

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution
+
+import java.nio.charset.StandardCharsets
+import java.sql.{Date, Timestamp}
+
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.catalyst.util.DateTimeUtils
+import org.apache.spark.sql.execution.command.{DescribeTableCommand, ExecutedCommandExec, ShowTablesCommand}
+import org.apache.spark.sql.internal.SQLConf
+import org.apache.spark.sql.types._
+
+/**
```

Comment:
Hm, it's just moving Hive stuff from `sql` to `sql`. If it were getting rid of it completely or moving it to test scope only, then I'm supportive but now it just move the function.

Kind of neutral on this. Code itself looks good. If any committer pushes it within few days, then it's good. Otherwise, I would close.

Reply:
> Hm, it's just moving Hive stuff from sql to sql.

This is not just moving it inside of sql. The idea is to clean up common class `QueryExecution` from pretty specific method which is used in tests mostly (and in `SparkSQLDriver` unfortunately). 

> If it were getting rid of it completely or moving it to test scope only, then I'm supportive but now it just move the function.

I tried all those variants like:
1. Moving to `sql/core/test` but it seems weird that `hive-thriftserver` (`SparkSQLDriver`) depends on a test jar.
2. Moving to `hive` since hive's tests use the method makes cycle dependencies `hive` <-> `sql/core`.

It would be nice if `SQLQueryTestSuite` doesn't depend on Hive related method. Is it really needed? 

> Otherwise, I would close.

@hvanhovell Please, protect ;-)